src/bundle/EventListener/CsrfListener.php

```diff
@@ -143,6 +147,8 @@ protected function isLoginRequest($route)
      * @param string $route
      *
      * @return bool
+     *
+     * @deprecated since Ibexa DXP 3.3.7. Add csrf_protection: false attribute to route definition instead.
```

Comment:
Can we use `trigger_deprecation` from Symfony contracts to log situations where `isSessionRoute` returns true? Does it make sense?